packages/db/src/lib/granule.ts

```diff
@@ -354,3 +360,61 @@ export const getGranulesByGranuleId = async (
     .where({ granule_id: granuleId });
   return records;
 };
+
+/**
+ * Change a granules' PG record and its files' PG record based on collection move
```

Comment:
Nit: formatting.      "Change granules PG record and files" or something like that.

Reply:
We need to better define what 'change' means.    Is this an upsert, a bounded modification, etc.